l10n_br_account/models/account_invoice.py

```diff
@@ -43,6 +43,7 @@
 
 class AccountInvoice(models.Model):
     _inherit = 'account.invoice'
+    _order = "internal_number desc, id desc"
```

Comment:
@sadamo why you add _order property in l10n_br_account and l10n_br_account_product, I can add _order property only in l10n_br_account is enough.